esmvaltool/diag_scripts/primavera/land_atmos_interactions.py

```diff
@@ -0,0 +1,179 @@
+import os
+import logging
+
+import iris
+import iris.cube
+import iris.analysis
+from iris.analysis import stats
+import iris.util
+
+from scipy import signal
+
+import esmvaltool.diag_scripts.shared
+import esmvaltool.diag_scripts.shared.names as n
+from esmvaltool.diag_scripts.shared import group_metadata
+
+from esmvalcore.preprocessor._regrid_esmpy import regrid
```

Comment:
Please only import public functions, e.g. from `esmvalcore.preprocessor import regrid, mask_landsea, extract_season`. Functions starting with an underscore should be considered private and may be renamed/removed/changed without notice, see [PEP8](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces) for more info.

Reply:
I'm still working on that one!!

Reply:
Sorry, I mentioned it because something similar happened in the other pull request I just reviewed.